app/src/test/java/com/codesoom/assignment/utils/JwtUtilTest.java

```diff
@@ -0,0 +1,42 @@
+package com.codesoom.assignment.utils;
+
+import com.codesoom.assignment.utils.JwtUtil;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.SignatureException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class JwtUtilTest {
+
+    private static final String SECRET = "12345678901234567890123456789010";
+    private static final String VALID_TOKEN = "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOjF9.neCsyNLzy3lQ4o2yliotWT06FwSGZagaHpKdAkjnGGw";
+    private static final String INVALID_TOKEN = "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOjF9.neCsyNLzy3lQ4o2yliotWT06FwSGZagaHpKdAkjnGGK";
```

Comment:
```suggestion
    private static final String VALID_TOKEN = "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOjF9.neCsyNLzy3lQ4o2yliotWT06FwSGZagaHpKdAkjnGGw";
    private static final String INVALID_TOKEN = VALID_TOKEN + "WRONG";
```

이렇게 표현할 수도 있을 것 같아요.
혹은 이런 토큰들은 값만 봐서는 검증이 어렵기 때문에 테스트에서는 그냥 믿고 사용하는 데이터 입니다. 그래서 fixture로 분리해도 좋을 것 같아요